gst/nnstreamer/tensor_filter/tensor_filter_single.c

```diff
@@ -237,6 +242,33 @@ g_tensor_filter_allocate_in_invoke (GTensorFilterSingle * self)
   return spriv->allocate_in_invoke;
 }
 
+/**
+ * @brief Set invoke async callback function for async invoke
+ * @param self "this" pointer
+ * @param callback callback function to be called when output is ready
+ * @param async_handle handle to be passed to the callback function
+ */
+static gboolean
+g_tensor_filter_single_set_invoke_async_callback (GTensorFilterSingle * self,
+    invoke_async_callback callback, void *handle)
+{
+  GTensorFilterSinglePrivate *spriv;
+  GstTensorFilterPrivate *priv;
+
+  g_return_val_if_fail (self != NULL, FALSE);
+  g_return_val_if_fail (callback != NULL, FALSE);
+
+  spriv = G_TENSOR_FILTER_SINGLE_PRIV (self);
+  g_return_val_if_fail (spriv != NULL, FALSE);
+
+  priv = &spriv->filter_priv;
+  g_return_val_if_fail (priv && &priv->prop != NULL, FALSE);
```

Comment:
this is always true, unnecessary condition.

Reply:
Thank you. I've updated it.